TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -597,12 +597,11 @@
 				}
 
 				.neos-button-small {
-					height: 32px;
-					width: 32px;
-					padding: 0 10px;
+					height: $wideMargin;
+					width: $wideMargin;
```

Comment:
it's simply because the whole UI is build around these lengths so it looks conformed, have you seen https://docs.google.com/document/d/1zQEcBHfcLeU1qg0Ws1E8a10wBf-_b5ehCVxr_PKRZgw/edit?